test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -444,3 +444,27 @@ def test_jobid_duplication(self):
         # Job should submit successfully because all existing id's has been
         # passed
         self.submit_job(lower=1, upper=2, job_id='3[]')
+
+    def test_jobid_resvid_after_multiple_restart(self):
+        """
+        Test to check the Jobid/Resvid should not wrap to 0 during
+        server restart multiple times consecutively either gracefully/abruptly
+        """
+        a = {'resources_available.ncpus': 15}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        self.set_svr_sv_jobidnumber(0)
+        self.submit_job(job_id='0')
+        self.submit_job(lower=1, upper=2, job_id='1[]')
+        self.submit_resv(resv_id='R2')
+        # Gracefully stop and start the server twice consecutively
+        self.stop_and_restart_svr('normal')
+        self.stop_and_restart_svr('normal')
+        self.submit_job(job_id='3')
+        self.submit_job(lower=1, upper=2, job_id='4[]')
+        self.submit_resv(resv_id='R5')
+        # Abruptly kill and start the server twice consecutively
```

Comment:
Do we really need to check the case of abrupt killing of server as well ? In this case whatever value gets saved at the timing of abrupt kill of server, the same value gets recovered while server is getting restarted.

Reply:
Yes it's important @suresh-thelkar. Now this functionality has been changed, after abruptly kill the server user will not get the next jobid/resvid.